storage/include/rocksdb/native_client.h

```diff
@@ -163,6 +163,11 @@ class NativeClient : public std::enable_shared_from_this<NativeClient> {
   ::rocksdb::ColumnFamilyHandle *defaultColumnFamilyHandle() const;
   ::rocksdb::ColumnFamilyHandle *columnFamilyHandle(const std::string &cFamily) const;
 
+  void createCheckpointNative(const uint64_t &checkPointId);
```

Comment:
Why do we need to define this functionality in `NativeClient` is all it does is calling the `IDBClient` interface?

Reply:
This is added only for gtest